### ISSUE TITLE
Having Client.py create it's own Logging handler causes duplicate logs.

### ISSUE THREAD

**Issue by gitmeister** [eXamadeus-zz/godaddypy#23]
In the app that uses Godaddypy I currently have: 
`logging.basicConfig(filename='logfile.log', format='%(asctime)s %(message)s', level=logging.DEBUG)`

When enabling the Client.py logging with:
`userClient = Client(userAccount,log_level=logging.DEBUG)`

I get all the logs showing in my log file (via propagation since by logger is 'root') but I also get Client.py duplicated on the console (via stderr) because it also has it's own separate StreamHandler.

Maybe i'm not understanding how Logging works but the intent is to have all the logs appear in a log file I created and not have some appear on the console. In a pinch I guess I can just do 2>/dev/null.

**Comment by eXamadeus-zz** [comment]
Thanks for that info!  This was my first forray into Python, so I didn't quite understand how logging propagation worked.  Adding my own StreamHandler seems to be a big no-no.

I have modified the library to have a better propagation standard using http://pythonsweetness.tumblr.com/post/67394619015/use-of-logging-package-from-within-a-library as a guide.

Here is an example of it working now:

```
>>> from godaddypy import Client, Account
>>> import logging
>>> logger = logging.getLogger()
>>> logger.setLevel(logging.INFO)
>>> logger.addHandler(logging.StreamHandler())
>>> client=Client(Account(api_key='REDACTED',api_secret='REDACTED'))
>>> client.get_domains()
[u'juliancoy.com', u'r00ted.info', u'r00ted.us']
>>> client=Client(Account(api_key='REDACTED',api_secret='REDACTED'), log_level=logging.DEBUG)
>>> client.get_domains()
[GET] response: <Response [200]>
Response data: [...,"domain":"juliancoy.com","status":"ACTIVE",...},{...,"domain":"r00ted.info","status":"ACTIVE",...},{...,"domain":"r00ted.us","status":"CANCELLED",...}]
Discovered domains: juliancoy.com
Discovered domains: r00ted.info
Discovered domains: r00ted.us
[u'juliancoy.com', u'r00ted.info', u'r00ted.us']
```

versus what was there:

```
>>> from godaddypy import Client, Account
>>> import logging
>>> logger = logging.getLogger()
>>> logger.setLevel(logging.INFO)
>>> logger.addHandler(logging.StreamHandler())
>>> client=Client(Account(api_key='REDACTED',api_secret='REDACTED'), log_level=logging.DEBUG)
>>> client.get_domains()
[GET] response: <Response [200]>
[GET] response: <Response [200]>
Response data: [{...,"domain":"juliancoy.com","status":"ACTIVE",...},{...,"domain":"r00ted.info","status":"ACTIVE",...},{...,"domain":"r00ted.us","status":"CANCELLED",...}]
Response data: [{...,"domain":"juliancoy.com","status":"ACTIVE",...},{...,"domain":"r00ted.info","status":"ACTIVE",...},{...,"domain":"r00ted.us","status":"CANCELLED",...}]
Discovered domains: juliancoy.com
Discovered domains: juliancoy.com
Discovered domains: r00ted.info
Discovered domains: r00ted.info
Discovered domains: r00ted.us
Discovered domains: r00ted.us
[u'juliancoy.com', u'r00ted.info', u'r00ted.us']
```

I'll check in the code and release v 2.2.3

**Comment by eXamadeus-zz** [comment]
Closing as issue is fixed in 2.2.3

**Comment by gitmeister** [comment]
Works great, thanks!
